Langs/0401.lang

```diff
@@ -0,0 +1,59 @@
+[MetaData]
+Language=العربية - المملكة المغربية
```

Comment:
you gotta duplicate the same file for all the instants of Arabic Language in Windows, there should be no difference in the file excluding this line

Reply:
Well If this file gets approved let's make another one for Saudi and others but I think just one is fine!

Reply:
But I don't think it's needed since Arabic is Arabic we don't want to include our accents, do we?